src/ComponentBase.ts

```diff
@@ -153,20 +152,20 @@ export abstract class ComponentBase<P extends {}, S extends _.Dictionary<any>> e
 
     // Subclasses may override, but _MUST_ call super.
     componentWillUnmount(): void {
-        _.forEach(this._handledSubscriptions, (subscription: StoreSubscriptionInternal<P, S>) => {
-            this._cleanupSubscription(subscription);
-        });
+        Object
+            .keys(this._handledSubscriptions)
+            .forEach(key => this._cleanupSubscription(this._handledSubscriptions[key]));
```

Comment:
Suggestion: `values(this._handledSubscriptions).forEach(sub => this._cleanupSubscription(sub))`